app/state_machines/efile_submission_state_machine.rb

```diff
@@ -39,8 +39,11 @@ class EfileSubmissionStateMachine
   after_transition(to: :preparing) do |submission|
     fraud_indicator_service = FraudIndicatorService.new(submission.client)
     hold_indicators = fraud_indicator_service.hold_indicators
-    if hold_indicators.present? && !submission.admin_resubmission?
+    hold_all = !ENV["FRAUD_HOLD_EVERYTHING"].blank?
```

Comment:
my favorite env-to-boolean trick: 
```suggestion
    hold_all = ActiveModel::Type::Boolean.new.cast(ENV["FRAUD_HOLD_EVERYTHING"])
```

It will cast all the usual suspects: https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html

Reply:
👍  thanks!